src/api/features/features.api.js

```diff
@@ -387,10 +387,15 @@ async function identifyOnExternalWmsLayer(config) {
                     coordinates: [...coordinate],
                 }
             }
+            let featureId = feature.id ?? feature.properties?.id
+            // if no feature ID defined, we take the first property and use it as an "ID"
+            if (!featureId && feature.properties) {
+                featureId = feature.properties[Object.keys(feature.properties)[0]]
+            }
```

Comment:
Here I would take the following as id and nothing (`-`) if none of those are available:
- id
- identifier
- name
- title

The search of those properties should be done case insensitive (id, Id, ID, ...)

Reply:
There are cases (like our backend... :see_no_evil: ) which doesn't give ANY property that contains ID, but the first one in the "list of properties" is a decent identifier.

So I can start by looking if anything looks like an ID (contains some `.id` or is `id` case insensitive) and if really nothing was found default to the first prop.

Reply:
ok with the fallback to the first prop if no id, identifier, name or title is found.